app/views/posts/show.html.erb

```diff
@@ -1 +1,16 @@
-<h1> Post Page</h1>
\ No newline at end of file
+<h1> Post Page</h1>
+<div>
+  <div>
+    <span>
+      <%= @post.title %> by <%= @user.name%>
+    </span>
+    <span>
+      <span >Comments: <%= @post.comments_counter %></span>
+      <span >Likes: <%= @post.likes_counter %></span>
+    </span>
+  </div>
+  <div>
+    <%= @post.text %>
+  </div>
+  <%= render partial: "shared/commentscard", locals: { comments: @comments}%>
+</div>
```

Comment:
- [ ] Please edit the post show view design to be like the given design and kindly ensure that you add all elements in the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/180583287-8ac04cf6-5029-4b8f-809d-b2917b4f97f0.png)  | ![image](https://user-images.githubusercontent.com/68796232/180583296-80a421ca-98fb-477d-a6e8-7af2c0373cb9.png)  |